src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.onKeyUpHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.onKeyUpHandler);
+  }
+
+  onKeyUpHandler = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
```

Comment:
try using `event.code` to check if the pressed key is a `SPACE` and write `SPACE` if so 😉 
<img width="185" alt="image" src="https://user-images.githubusercontent.com/55802630/182562057-89c74773-84ee-490a-92d5-6e1b783384c0.png">